Release Notes for 1.15.2 (#12055)
istio/istio.io: content/en/news/releases/1.15.x/announcing-1.15.2/index.md

---
title: Announcing Istio 1.15.2
linktitle: 1.15.2
subtitle: Patch Release
description: Istio 1.15.2 patch release.
publishdate: 2022-10-11
release: 1.15.2
---

This release includes security fixes in Go 1.19.2 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
This release contains bug fixes to improve robustness.

> This release note describes what is different between Istio 1.15.2 and Istio 1.15.2.

Reviewer: ```suggestion
This release note describes what is different between Istio 1.15.1 and Istio 1.15.2.
```

{{< relnote >}}

## Changes

> - **Fixed** an issue that default `idleTimeout` for passthrough cluster has been changed to `0s` since 1.14.0, disabling the timeout. Restored previous behavior to using Envoy's default value of 1 hour. ([Issue #41114](https://github.com/istio/istio/issues/41114))

Reviewer: Thanks. Your update makes more sense. 
```suggestion
- **Fixed** an issue that the default `idleTimeout` for the passthrough cluster was changed to `0s` in 1.14.0, disabling the timeout. Restored the previous behavior to using Envoy's default value of 1 hour. ([Issue #41114](https://github.com/istio/istio/issues/41114))
```

> - **Fixed** the `gateway-api` integration to read `v1beta1` resources for `HTTPRoute`, `Gateway`, and `GatewayClass`. Users of the `gateway-api` must be on version 0.5.0 or higher before upgrading Istio.

Reviewer: Can we just remove the last sentence `Users of the `gateway-api` must be on version 0.5.0 or higher before upgrading Istio.`? https://istio.slack.com/archives/C036AKKFZ7Z/p1665081516268089 is a pointer to were I brought this up.

Reviewer: The 1.13.9 RN (https://github.com/istio/istio.io/pull/12053) as well as the 1.14.5 RN changed this to `Fixed the gateway API integration to not fail when the v1alpha2 version is removed.`.

---

> title: Announcing Istio 1.15.1

Reviewer: Version is wrong in a four places..

linktitle: 1.15.1
subtitle: Patch Release
description: Istio 1.15.1 patch release.

> publishdate: 2022-09-26

Reviewer: Date is wrong.

release: 1.15.1
---

This release includes security fixes in Go 1.19.2 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.

This release contains bug fixes to improve robustness.

> 

Reviewer: Need to verify if we want these lines as separate lines. In earlier release notes, we haven't (ex: https://istio.io/latest/news/releases/1.14.x/announcing-1.14.3/)

This release note describes what is different between Istio 1.15.1 and Istio 1.15.2.

{{< relnote >}}

## Changes

> - **Fixed** an issue that default `idleTimeout` for passthrough cluster has been changed to `0s` since 1.14.0 and timeout is disabled. Previous behavior is using envoy's default value. ([Issue #41114](https://github.com/istio/istio/issues/41114))

Reviewer: ```suggestion
- **Fixed** an issue where the default `idleTimeout` for the passthrough cluster has been changed to `0s` since 1.14.0 and the timeout is disabled. Previous behavior was using Envoy's default value. ([Issue #41114](https://github.com/istio/istio/issues/41114))
```

I'm not really sure what this fix is about. We mention pre 1.14.0 we used Envoy's default value.  In 1.14.0 the default is 0 and the timeout is disabled. Now we fixed it, but what did we fix it to?

Author: If I understand the linked issue correctly, the default in Envoy is 1h and 0s disables the timeout... so we no longer use 0s as the timeout. I'm guessing, the release note is not clear.

> - **Fixed** the gateway-api integration to read `v1beta1` resources for `HTTPRoute`, `Gateway`, and `GatewayClass`. Users of the `gateway-api` must be on version 0.5.0 or higher before upgrading Istio.

Reviewer: Should we tick the first gateway-api as well to be consistent?

Reviewer: Should there be an upgrade note for this? Or is this enough notification being in the release notes?
